packages/data-stores/src/starter-designs-queries/use-starter-designs-query.ts

```diff
@@ -0,0 +1,101 @@
+import { stringify } from 'qs';
+import { useQuery, UseQueryResult, QueryOptions } from 'react-query';
+import wpcomRequest from 'wpcom-proxy-request';
+import type { StarterDesigns } from './types';
+import type { Category, Design, DesignRecipe } from '@automattic/design-picker/src/types';
+
+interface StarterDesignsQueryParams {
```

Comment:
Thanks for these typings changes, looks much cleaner now!! 👍

Reply:
thanks for suggesting, yes it looks much better!